packages/app-runtime/test/lib/natives/FakeNativeNotificationAccess.ts

```diff
@@ -1,8 +1,8 @@
 import { ILogger } from "@js-soft/logging-abstractions";
 import { Result } from "@js-soft/ts-utils";
-import { INativeNotificationAccess, INativeNotificationScheduleOptions } from "../../src";
+import { INativeNotificationAccess, INativeNotificationScheduleOptions } from "../../../src";
 
-export class NativeNotificationAccessMock implements INativeNotificationAccess {
+export class FakeNativeNotificationAccess implements INativeNotificationAccess {
```

Comment:
I think you forgot to change some log entries regarding the renaming of `NativeNotificationAccessMock` to `FakeNativeNotificationAccess`.